docker/pypi/wmagent/init.sh

```diff
@@ -415,6 +421,10 @@ agent_tweakconfig() {
             GLOBAL_DBS_URL=https://cmsweb-testbed.cern.ch/dbs/int/global/DBSReader
             sed -i "s+DBSInterface.globalDBSUrl = 'https://cmsweb.cern.ch/dbs/prod/global/DBSReader'+DBSInterface.globalDBSUrl = '$GLOBAL_DBS_URL'+" $WMA_CONFIG_DIR/config.py
             sed -i "s+DBSInterface.DBSUrl = 'https://cmsweb.cern.ch/dbs/prod/global/DBSReader'+DBSInterface.DBSUrl = '$GLOBAL_DBS_URL'+" $WMA_CONFIG_DIR/config.py
+            rucio_host=http://cms-rucio.cern.ch
+            rucio_auth=https://cms-rucio-auth.cern.ch
+            sed -i "s+WorkQueueManager.rucioUrl =.*+WorkQueueManager.rucioUrl+WorkQueueManager.rucioUrl = $rucio_host'+" $WMA_CONFIG_DIR/config.py
```

Comment:
You have one item extra in this sed command, in addition to be missing a single quote. The command you wanted is likely: `sed -i "s+WorkQueueManager.rucioUrl =.*+WorkQueueManager.rucioUrl = '$rucio_host'+" $WMA_CONFIG_DIR/config.py`

But please make sure to test these out

Reply:
thanks. fixed